frontend/ios/fastlane/Fastfile

```diff
@@ -92,6 +92,8 @@ platform :ios do
        output_directory: "#{ENV['HOME']}",
        output_name: "#{build_config_name}.ipa",
        export_method: "app-store",
+       # pass base64 encoded env variable "environment=production" directly, because shell result does not work for gym as it does for gradle (android/Fastfile)
```

Comment:
I think this is not correct, we did successful releases with passing vars like this.

Reply:
You can test the ipa without and it won't work so no idea what happened meanwhile. I mean why we set this for gradle and not for gym?